skbio/metadata/_intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
I'd prefer not to have it, since that is additional code we must maintain and we are essentially forking the bx-python code without a clear use-case. If it is needed in the future, we have a reference here to the code. That being said, I don't have a very strong opinion so I will defer to you guys.